docs/content/mobile/ios/cloud-deploy.md

```diff
@@ -94,9 +94,9 @@ To do this:
 1. In the same drop-down window, ensure that the checkbox `ShouldConnectLocally` is not checked.
 1. In your Unity Editor, navigate to **SpatialOS** > **Build for cloud**. Select your iOS client-worker, and wait for the build to complete. <br/>
 You know it’s complete when it says `Completed build for Cloud target` in your Unity Editor’s Console window.
-1. Select **SpatialOS** > **Launch mobile client** > **iOS Device**.
-1. In your file manager, navigate to `/workers/unity/build/worker/` and locate the `.xcodeproj` that corresponds to your iOS client-worker, it may be in a sub-folder.<br>
-	Open it in Xcode.
+1. In your file manager, navigate to `/workers/unity/build/worker/iOSClient@iOS`.<br>
```

Comment:
also need to state that this is from the root of your SpatialOS project

Reply:
fixed, thanks